src/main/java/game/Magic.java

```diff
@@ -17,7 +17,7 @@ public Magic(long mask, long factor, int offset) {
     }
 
     // Precomputed overlapping fixed shift magics:
-    // https://en.wikipedia.org/wiki/Bitboard#Magic_bitboards
+    //  https://www.chessprogramming.org/Magic_Bitboards
```

Comment:
Could you please change to https://www.chessprogramming.org/Magic_Bitboards#Fixed_shift_Fancy (that's the specific appraoch that's used here) and remove the double leading space?